vignettes/data_protocol.Rmd

```diff
@@ -76,15 +69,27 @@ The distributions currently included in the literature search for each pathogen/
 - "ebola serial interval"
 - "influenza H7N9 onset to admission"
 
+However, these simple search phrases can return a large number of irrelevant papers. We recommend using more specific search schema depending on the search engine used. For example, if using Google Scholar a schema like:
+
+- ("Middle East Respiratory Syndrome" OR MERS) AND "onset to death" AND (estimation OR inference OR calculation)
+- (ebola OR EVD) AND "onset to death" AND (estimation OR inference OR calculation)
+
+Or if Web of Science is being used:
+
+- ("COVID-19" OR "SARS-CoV-2") AND "onset to death" AND estimat*
```

Comment:
Here I would translate the search from lines 69 and 70 to their equivalents in google scholar/WoS, rather than changing it to other diseases and parameters, so people can see exactly how its done

Reply:
Use MERS and Ebola in the WoS example to make it clearer what the difference between scholar and WoS is.